server/index.js

```diff
@@ -9,10 +9,10 @@ const PORT = 3000;
 
 app.use('/', express.static(path.join(__dirname, '../public'))); //rooms/:id
```

Comment:
You can change the '/' to '/rooms/:id' to be the same format as everyone.

Reply:
This is now updated to be what the group has talked about so now when putting in localhost/rooms/(any number here) it only pools those reviews with the matching houseId on my schema. Thank you as well for the help implementing that!